Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "node-bindgen"
-version = "4.2.2"
+version = "4.2.3"
```

Comment:
There is no need to bump up this since only dependent have changed

Reply:
Got it, I'll change that back